Makefile

```diff
@@ -6,7 +6,7 @@ Q1 = $(V:1=)
 Q = $(Q1:0=@)
 ECHO1 = $(V:1=@ :)
 ECHO = $(ECHO1:0=@ echo)
-FUZZ_OUTPUT_DIR = $(shell pwd)/fuzz/output
+FUZZ_OUTPUT_DIR = $(CURDIR)/fuzz/output
```

Comment:
`CURDIR` is technically different from `shell pwd` in that the latter isn’t stuck constant if the `Makefile` somehow `cd`s halfway in the script.
That said, do we really want the build to base on the working directory rather than fixed at the repo root?